### PR TITLE
fix(ci): fix flamegraph generation for rewritten queries

### PR DESCRIPTION
**Pull request by kwannoel**
I hereby agree to the terms of the [RisingWave Labs, Inc. Contributor License Agreement](https://gist.github.com/TennyZhuang/f00be7f16996ea48effb049aa7be4d66#file-rw_cla).

## What's changed and what's your intention?

Some queries left out of flamegraph generation because of bad regex / not synced with kube-bench sink templates.

Fixes q6 as well to use blackhole sink.

## Checklist

- [ ] I have written necessary rustdoc comments
- [ ] I have added necessary unit tests and integration tests
- [ ] I have added fuzzing tests or opened an issue to track them. (Optional, recommended for new SQL features #7934).
- [ ] My PR contains breaking changes. (If it deprecates some features, please create a tracking issue to remove them in the future).
- [ ] All checks passed in `./risedev check` (or alias, `./risedev c`)
- [ ] My PR changes performance-critical code. (Please run macro/micro-benchmarks and show the results.)

- [ ] My PR contains critical fixes that are necessary to be merged into the latest release. (Please check out the [details](https://github.com/risingwavelabs/risingwave/blob/main/CONTRIBUTING.md))

## Documentation

- [ ] My PR contains user-facing changes.

<details><summary>Click here for Documentation</summary>

### Types of user-facing changes

Please keep the types that apply to your changes, and remove the others.

- Installation and deployment
- Connector (sources & sinks)
- SQL commands, functions, and operators
- RisingWave cluster configuration changes
- Other (please specify in the release note below)

### Release note

</details>

### CODE REVIEW

File: ci/scripts/sql/nexmark/q19.sql
Comment (stdrc) [inline ci/scripts/sql/nexmark/q19.sql:1]: Would suggest renaming this to `q19-no-output`, cuz it's not original q19.

File: ci/scripts/sql/nexmark/q19.sql [hunk 1]
```diff
@@ -0,0 +1,12 @@
+-- noinspection SqlNoDataSourceInspectionForFile
+-- noinspection SqlResolveForFile
+CREATE SINK nexmark_q19 AS
+SELECT auction, bidder, price, channel, url, date_time, extra
```
Comment (stdrc) [inline ci/scripts/sql/nexmark/q19.sql:4]: trailing white space🧐

File: ci/scripts/sql/nexmark/q5-many-windows.sql [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            window_start AS starttime_c
+		FROM
+            HOP(bid, date_time, INTERVAL '5' SECOND, INTERVAL '5' MINUTE)
+        GROUP BY
+            bid.auction,
+            window_start
+		) AS CountBids
+	GROUP BY
+        CountBids.starttime_c
+	) AS MaxBids
```
Comment (stdrc) [inline ci/scripts/sql/nexmark/q5-many-windows.sql:39]: mixed tabs and spaces🤪

**Review by stdrc — APPROVED** [review]
Rest looks great to me!

**Review by lmatz — APPROVED** [review]
LGTM

File: ci/scripts/sql/nexmark/q19.sql (hunk 1 above)
Comment (kwannoel) [reply ci/scripts/sql/nexmark/q19.sql:4]: Extracted directly from kube-bench via https://github.com/risingwavelabs/kube-bench/blob/main/scripts/extract-queries.sh, so won't fix in this PR. Need to patch it in kube-bench repo.

File: ci/scripts/sql/nexmark/q19.sql
Comment (kwannoel) [reply ci/scripts/sql/nexmark/q19.sql:1]: As per https://github.com/risingwavelabs/risingwave/pull/10976#discussion_r1265435236

File: ci/scripts/sql/nexmark/q5-many-windows.sql (hunk 2 above)
Comment (kwannoel) [reply ci/scripts/sql/nexmark/q5-many-windows.sql:39]: As per https://github.com/risingwavelabs/risingwave/pull/10976#discussion_r1265435236

File: ci/scripts/sql/nexmark/q19.sql
Comment (kwannoel) [reply ci/scripts/sql/nexmark/q19.sql:1]: https://github.com/risingwavelabs/kube-bench/blob/43c8777ecf38ca9fa137d90886bed85052d1818a/manifests/nexmark/nexmark-sinks.template.yaml#L410-L422

Is there some original q19.sql? It does not seem to exist anymore..

**Comment by kwannoel** [comment]
Only left q5 not passing, but it's just due to buildkite cancelling the job.

**Pull request merged**